public/js/settings/conditions.js

```diff
@@ -258,18 +258,18 @@ pimcore.bundle.personalization.settings.conditions = (function () {
                         var leafletMap, marker, circle;
 
                         var searchHandler = function() {
-                            var address = searchfield.getValue();
-                            Ext.Ajax.request({
-                                url: pimcore.bundle.personalization.settings.conditions.getSearchUrl(address),
-                                method: "GET",
-                                success: function (response, opts) {
-                                    var data = Ext.decode(response.responseText);
+                            const address = searchfield.getValue();
+                            pimcore.helpers.sendRequest(
+                                "GET",
+                                pimcore.settings.targeting.conditions.getSearchUrl(address),
```

Comment:
```suggestion
                                pimcore.bundle.personalization.settings.conditions.getSearchUrl(address),
```

Reply:
thank you, nice catch! 
I didn't noticed it when copy/paste from 10.6 🤦🏻

Reply:
No problem :)